mesonbuild/mdist.py

```diff
@@ -284,4 +283,5 @@ def run(options):
     if rc == 0:
         for name in names:
             create_hash(name)
+            print('Created', os.path.relpath(name))
```

Comment:
Printing that it is *created*, is indeed more sensible than printing the bare filename + hash.